playbooks/configure-services.yml

```diff
@@ -16,11 +16,19 @@
     service: name=NetworkManager enabled=no
     ignore_errors: yes
 
+  - name: Start network manager service
+    service: name=NetworkManager state=started
+
+  - name: Enable network manager service
+    service: name=NetworkManager enabled=yes
+
   - name: Start network service
     service: name=network state=started
+    ignore_errors: yes
```

Comment:
what if it failed to start? error will be ignored, will not it cause issue?

Reply:
so from rhel8, network service, is not used, it is handled by Network Manager service, 
unless otherwise configured.
so failure to start is expected. going forward this entire network service will be removed, and only network manager will be there

Reply:
What if same issue occured in RHEL7?

Reply:
@gobindadas no this is in gdeploy -3.0.0 , for RHEL8 only.

for RHEL7- gdeploy 2.00 (i.e master branch will be used)